src/wallet/wallet2.cpp

```diff
@@ -2376,6 +2419,12 @@ bool wallet2::check_connection(uint32_t *version, uint32_t timeout)
 
   boost::lock_guard<boost::mutex> lock(m_daemon_rpc_mutex);
 
+  // TODO: Add light wallet version check.
+  if(m_light_wallet) {
+      version = 0;
```

Comment:
This is the same as `version = nullptr;` which is of no use for the caller. Shouldn't it be `*version = 0;` instead?